### PR TITLE
Implemented the changes: Return dictionary

### PR DESCRIPTION
**Pull request by iABn0rma1**
## Description

> Now returns a list of dictionary for easy readability, and removed CSV conversion.

## Checklist

> Before submitting this pull request, kindly verify that the ensuing checkpoints have been reached.

- [x] Have you adhered to the repository's defined coding convention rules?
- [x] Have you updated the 'documentation.md' file with the method/function documentation?
- [x] Have you sent a message along with the result or response?
- [ ] Have you used the try-catch technique?
- [x] Has the method/class been added to the documentation (md file)?

## Screenshots

image

## Additional Notes/Comments

> Please let me know if any other changes are required.

---

I certify that I have carried out the relevant checks and provided the requisite screenshot for validation by submitting this pull request.
I appreciate your contribution.

### CODE REVIEW

File: src/scrape_up/steam/steamScraper.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+        Returns:
+            list: List of dictionaries with game data.
+        """
+        print(data)
+        readable_data = []
+        for i in range(len(data["Name"])):
+            game_data = {col: data[col][i] for col in self.cols}
+            readable_data.append(game_data)
+        return readable_data
```
Comment (nikhil25803) [inline src/scrape_up/steam/steamScraper.py:206]: Have you tested the code, getting the Error - 
```bash
UnicodeEncodeError: 'charmap' codec can't encode character '\u20b9' in position 532: character maps to <undefined>
```

Also, upload Screenshot of output from the next time.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/steam/steamScraper.py (hunk 1 above)
Comment (iABn0rma1) [reply src/scrape_up/steam/steamScraper.py:206]: Yes, I tested the code by scraping 300 entries for each filter.
Output Screenshot: 
image

Also I apologies for not including it previously, as it only removed the conversion to pandas data frame and CSV. As addressed in #949

**Review by nikhil25803 — APPROVED** [review]

**Comment by nikhil25803** [comment]
Great work @iABn0rma1  !!
Do not forget to
+ Share your contribution on socials and tag Clueless 🫂
+ Give this repository a star ✨
+ Follow Clueless Community on GitHub ✅

**Comment by iABn0rma1** [comment]
🚀 Thank you so much, @nikhil25803 ! 

🌟 Already starred the repo and following Clueless Community.
🤝 I'm excited to share this contribution with the community, will definitely tag Clueless on socials.! 

I'd like to contribute more by optimising the code:
- Multi-filter_tags scraping: To process multiple tags concurrently, improving efficiency and avoiding potential duplicates.
image

- Adding exception handling.

**Pull request merged**